plugins/http_plugin/include/eosio/http_plugin/macros.hpp

```diff
@@ -13,7 +13,7 @@
            (const chain::next_function_variant<call_result>& result) mutable {                                  \
               if (std::holds_alternative<fc::exception_ptr>(result)) {                                          \
```

Comment:
Not related to your change, but instead of these `std::holds_alternative<.` and `std::get<>`, it would be cleaner to use `std::visit` with lambdas.